debian/control

```diff
@@ -224,11 +224,24 @@ Recommends: mir-demos,
             glmark2-es2,
             glmark2-es2-wayland,
             mesa-utils-extra,
-Description: Display Server for Ubuntu - stress tests and other test tools
+Description: Display Server for Ubuntu - test tools
  Mir is a display server running on linux systems, with a focus on efficiency,
  robust operation and a well-defined driver model.
  .
- Contains a tool for stress testing the Mir display server
+ Contains tools for smoke and performance testing the Mir display server
+
+Package: mir-wlcs-integration
+Architecture: linux-any
+Pre-Depends: ${misc:Pre-Depends}
+Breaks: mir-test-tools (<< 2.1)
```

Comment:
Hm. I'm reasonably certain that this is not going to be the right version, because this will matches the *new* package, too (so it'll break itself, and never resolve).

Because it's going to be difficult to get the version numbering right as it's dependent on when it's borsed, I'll fix this up and bors it.